modules/web/src/app/wizard/template.html

```diff
@@ -139,7 +145,7 @@
                      *ngIf="showCreateClusterButton"
                      icon="km-icon-add"
                      [label]="getPrimaryButtonLabel()"
-                     [disabled]="creating"
+                     [disabled]="creating || quotaExceededWarning"
```

Comment:
I found a edge case where user is allowed to create a cluster even if it exceeds and user will be able to create.

Steps: 
- navigate to cluster templates
- Lets suppose navigate to `Customize Template` option clusterTemplateID=XXXX&mode=customize-cluster-template
- Try to change MODE -> customize-cluster-template`XYZXYZ`
- Try reach end of cluster summary page

Update Initial node to e.g 20 replicase (Assuming: Quota WILL be exceeded for sure) it is not displaying SINCE quota widget is already hidden but it will be exceeded.
 
-  Button will show "Create Cluster" as Text and `ENABLED` to create Cluster.

IMO, this case should be handled and disallowed to create cluster

Reply:
@KhizerRehan maybe we can open an issue against that? I think the PR in its current state is fine and this is too much of an edge case.

That issue can help us evaluate if we need a fix in KKP 2.23 or afterward.

Reply:
Yes! it is not related to this PR. So i will create sep ticket for this.